test/unit/machine-learning/machine-learning-api-client.spec.ts

```diff
@@ -233,19 +288,38 @@ describe('MachineLearningApiClient', () => {
         .stub(HttpClient.prototype, 'send')
         .rejects(expected);
       stubs.push(stub);
-      return apiClient.createModel(NAME_ONLY_CONTENT)
+      return apiClient.createModel(NAME_ONLY_OPTIONS)
         .should.eventually.be.rejected.and.deep.include(expected);
     });
   });
 
   describe('updateModel', () => {
-    const NAME_ONLY_CONTENT: ModelContent = { displayName: 'name1' };
+    const NAME_ONLY_OPTIONS: ModelOptions = { displayName: 'name1' };
+    const GCS_OPTIONS: ModelOptions = {
+      displayName: 'name2',
+      tfliteModel: {
+        gcsTfliteUri: 'gcsUri1',
+      },
+    };
+    const AUTOML_OPTIONS: ModelOptions = {
+      displayName: 'name3',
+      tfliteModel: {
+        automlModel: 'automlModel',
+      },
+    };
+
     const NAME_ONLY_MASK = ['displayName'];
+    const GCS_MASK = ['displayName', 'tfliteModel.gcsTfliteUri'];
+    const AUTOML_MASK = ['displayName', 'tfliteModel.automlModel'];
 
-    const invalidContent: any[] = [null, undefined];
-    invalidContent.forEach((content) => {
-      it(`should reject when called with: ${JSON.stringify(content)}`, () => {
-        return apiClient.updateModel(MODEL_ID, content, NAME_ONLY_MASK)
+    const NAME_ONLY_UPDATE_MASK = "updateMask=displayName";
+    const GCS_UPDATE_MASK = "updateMask=displayName,tfliteModel.gcsTfliteUri";
```

Comment:
GCS_MASK_STRING to contrast with GCS_MASK

Reply:
That doesn't match the others. I renamed them all for further clarify.

Reply:
*clarity